### DISCUSSION TITLE
Contraints on PB in cpmg_ch3_mq

### DISCUSSION THREAD

**Discussion by OEnny09** [gbouvignies/ChemEx#291] [Q&A] [answered]
Hello! I am currently working on fitting cpmg_ch3_mq data, but I am encountering values that are much larger than they should be as well as large errors when it comes to PB. Is there a way to add constraints/an upper limit on what PB can be? If so, how would I go about doing that? Thank you!

**Comment by viochemist** [comment, marked as answer]
Yes, I believe you can. In the parameters file where you initialize PB you
can specify

PB = [0.1, 0.0, 0.5]

Which should initialize at 10% and limit 0-50%. Guillaume, is this still
correct?

**Comment by OEnny09** [comment]
@viochemist I just tried what you suggested and it seems to have worked. Thank you very much!
